Realm/ObjectStore/impl/apple/external_commit_helper.cpp

```diff
@@ -32,6 +32,87 @@
 using namespace realm;
 using namespace realm::_impl;
 
+#if TARGET_OS_TV
```

Comment:
In the future we should look to split the tvOS version of this into a separate file from the iOS + Mac implementation.